File: mAssignments/m1.java
package mAssignments;

import java.util.Scanner;

public class m1 {

    public void printUpto(int n) {
        for (int i = 1; i <= n; i++) {
            System.out.print(i * 2 + "\t");
        }
        System.out.println();
    }

    public static void main(String[] args) {

        int n;
        Scanner sc = new Scanner(System.in);
        System.out.println("Please enter a number: ");
        n = sc.nextInt();
        m1 m = new m1();
        m.printUpto(n);

    }
}

File: mAssignments/m1.cpp
#include <iostream>
using namespace std;

class printNumber
{
public:
    printNumber() { cout << "Constructor called.\n"; }

    void printUpto(int num)
    {
        for (int i = 1; i <= num; i++)
        {
            cout << i * 2 << "\t";
        }
        cout << "\n";
    }
};

int main()
{
    int n;
    cout << "Please enter the number: \n";

    cin >> n;
    printNumber p;
    p.printUpto(n);

    return 0;
}